aspnetcore/tutorials/razor-pages/razor-pages-start.md

```diff
@@ -273,17 +416,19 @@ At the end of this tutorial, you'll have a working Razor Pages web app that you'
 
   ![Home or Index page](razor-pages-start/_static/homeGDPR2.2.png)
 
-  The following image shows the app after you give consent to tracking:
+  The following image shows the app after consent to tracking is provided:
 
   ![Home or Index page](razor-pages-start/_static/home2.2.png)
   
 # [Visual Studio Code](#tab/visual-studio-code)
 
   [!INCLUDE[](~/includes/trustCertVSC.md)]
 
-* Press **Ctrl-F5** to run without the debugger.
+* Press **Ctrl+F5** to run without the debugger.
```

Comment:
The `<kbd>` tag is a better option for keyboard shortcuts to improve both styling and accessibility.

```suggestion
* Press <kbd>Ctrl+F5</kbd> to run without the debugger.
```